src/app/generate/generate.go

```diff
@@ -216,3 +236,18 @@ func tagVersionGetter(cCtx *cli.Context) (*git.TagsSource, error) {
 
 	return tvsrc, nil
 }
+
+func sanitizeValue(in []string) []string {
+	// The action leverages docker, but having no default the argument "--included-dirs=" causes
+	// to get an "empty" includeDir definition, but with "" as an element
+	// time="2023-05-24T15:25:42Z" level=error msg="includedDirs = [\"\"]"
+	// This causes "" to be passed as an element, that is translated to . and a slash is added becoming "./".
+	// We have an issue there since "./" matches all files on root, but not any of the folders that are specified as "folder1/..".
```

Comment:
nit: the comment is great for context but maybe is a bit too coupled with the way the issue was reproduced.
I wonder if we could include a test reproducing the issue and move the long explanation there. Here we could simply point out that `""` would become `"./"` with unexpected behavior.